src/rgw/rgw_admin.cc

```diff
@@ -4206,7 +4206,7 @@ int main(int argc, const char **argv)
        OPT::PUBSUB_NOTIFICATION_LIST,
 			 OPT::PUBSUB_TOPIC_GET,
        OPT::PUBSUB_NOTIFICATION_GET,
-       OPT::PUBSUB_TOPIC_STATS  ,
+       OPT::PUBSUB_TOPIC_STATS,
```

Comment:
probably best to revert these changes. may create merge issues later on without reason.